src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorArgumentDefinitions.java

```diff
@@ -217,4 +182,36 @@
          */
         CANONICAL
     }
+
+    public enum DataSourceType {
+        SIMPLE_XSV("simpleXSV"),
+        LOCATABLE_XSV("locatableXSV"),
+        GENCODE("gencode"),
+        COSMIC("cosmic");
```

Comment:
So in the future, we would add `VCF("vcf")`, etc etc?

Reply:
If so, please put a comment...

Reply:
Yes - it's analogous to the `type` field in the oncotator config files.